bindsnet/environment/__init__.py

```diff
@@ -121,27 +121,35 @@ def step(self, a: int) -> Tuple[torch.Tensor, float, bool, Dict[Any, Any]]:
 
         self.preprocess()
 
+        # Add the raw observation from the gym environment into the info
+        # for debugging and display
+        info['gym_obs'] = self.obs
+
         # Store frame of history and encode the inputs.
         if len(self.history) > 0:
             self.update_history()
             self.update_index()
-
-        raw_obs = self.obs
+            # add the delta observation into the info for debugging and
+            # display
+            info['delta_obs'] = self.obs
```

Comment:
Aren't you just storing `self.obs` twice?

Reply:
I pulled the history code that was inside of Pipeline previously. It overwrites `self.obs` internally. So the second will be the delta observation instead of the raw.